package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matterfront",
+  "version": "0.1.0",
+  "description": "Mattermost frontend app powered by electron",
+  "main": "./src/main.js",
+  "devDependencies": {
+    "electron-packager": "^5.1.1",
+    "electron-prebuilt": "^0.34.2"
+  },
+  "electronVersion":"0.34.1",
+  "scripts": {
+    "start":"electron ./",
```

Comment:
Ok I think I get it now (not used to node idioms), but that's probably my fault for having a `src` dir there. Feels weird to have this asymmetry between `app.asar` and `src`, making the config file lookup blow up in some cases since `app.getAppPath()` doesn't mean the same thing relative to `main.js`.